### PR TITLE
Max steps for adaptive integrators

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: ksp_plugin/plugin.hpp [hunk 1]
```diff
@@ -407,6 +407,7 @@ class Plugin {
     not_null<IndexToOwnedCelestial*> const celestials);
 
   Time const Δt_ = 10 * Second;
+  int const prolongation_max_steps_ = 1000;
```
Comment (eggrobin) [inline ksp_plugin/plugin.hpp]: `numeric_limits<int64_t>::max()`

Comment (eggrobin) [inline ksp_plugin/plugin.hpp]: Also, they should all be `int64_t`.

File: ksp_plugin/plugin.cpp [hunk 2]
```diff
@@ -969,6 +973,7 @@ void Plugin::SynchronizeNewVesselsAndCleanDirtyVessels() {
         history_time_,
         Ephemeris<Barycentric>::AdaptiveStepParameters(
             prolongation_integrator_,
+            prolongation_max_steps_,
             prolongation_length_tolerance_,
             prolongation_speed_tolerance_));
```
Comment (eggrobin) [inline ksp_plugin/plugin.cpp]: should be checked.

File: ksp_plugin/plugin.cpp [hunk 3]
```diff
@@ -1059,6 +1064,7 @@ void Plugin::EvolveProlongationsAndBubble(Instant const& t) {
         t,
         Ephemeris<Barycentric>::AdaptiveStepParameters(
             prolongation_integrator_,
+            prolongation_max_steps_,
             prolongation_length_tolerance_,
             prolongation_speed_tolerance_));
```
Comment (eggrobin) [inline ksp_plugin/plugin.cpp]: should be checked.

File: ksp_plugin/plugin.hpp (hunk 1 above)
Comment (pleroy) [inline ksp_plugin/plugin.hpp]: Done (2x).

File: ksp_plugin/plugin.cpp (hunk 2 above)
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: Done.

File: ksp_plugin/plugin.cpp (hunk 3 above)
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: Done.

**Comment by pleroy** [comment]
Fix #911.

**Pull request merged**